moped-editor/src/views/projects/projectView/KnackSync.js

```diff
@@ -1,205 +0,0 @@
-import React from "react";
```

Comment:
This file was the first implementation of sync ➡️  knack. It worked, but it lacked the updates that the component that is used in the summary page itself had received. As a result, it needed to be removed, as it was a ton of duplicated code and it allowed projects to be sync'd without consideration to signals and other recent updates.